readme.md

```diff
@@ -28,15 +28,16 @@ $ npm install open
 ```js
 const open = require('open');
 
-// Opens the image in the default image viewer
 (async () => {
-	await open('unicorn.png', {wait: true});
+	// Opens the image in the default image viewer
+	await open('unicorn.png', {wait: true}); 
+	// "wait" for the opened app to exit
```

Comment:
This is not correct, at this point, it's done waiting.

Reply:
moved comment to explain "wait" before the function call